Sources/CreateAPI/Generator/Generator.swift

```diff
@@ -87,16 +88,7 @@ final class Generator {
     }
     
     var fileHeader: String {
-        var output = ""
-        
-        if let comment = options.fileHeaderComment {
-            output += comment
-        } else {
-            output += """
-            // Generated by Create API
-            // https://github.com/CreateAPI/CreateAPI
-            """
-        }
+        var output = options.fileHeaderComment
```

Comment:
This was simplified because the default value is now defined in **ConfigOptions.swift**